src/YawPitchControl/YawPitchControl.js

```diff
@@ -634,33 +621,79 @@ export default class YawPitchControl extends Component {
 		}, duration);
 	}
 
-	get() {
-		return this.axes.get();
+	enableSensor() {
+		return new Promise((resolve, reject) => {
+			const activateSensor = () => {
+				if (this.options.gyroMode !== GYRO_MODE.NONE) {
+					this._deviceSensor.enable();
+					resolve();
+				} else {
+					reject(new Error("gyroMode not set"));
+				}
+			};
+
+			if (DeviceMotionEvent && typeof DeviceMotionEvent.requestPermission === "function") {
+				DeviceMotionEvent.requestPermission().then(permissionState => {
+					if (permissionState === "granted") {
+						activateSensor();
+					} else {
+						reject(new Error("Permission not granted"));
+					}
+				}).catch(e => {
+					// This can happen when this method was't triggered by user interaction
+					reject(e);
+				});
+			} else {
+				activateSensor();
+			}
+		});
 	}
 
-	getYaw() {
-		return this.axes.get().yaw;
+	disableSensor() {
+		this._deviceSensor.disable();
 	}
 
-	getPitch() {
-		return this.axes.get().pitch;
+	getYawPitch() {
+		const deviceYawPitchDelta = this._deviceSensor.getYawPitchDelta();
+
+		// Update axes values
+		this.axes.setBy({
+			yaw: deviceYawPitchDelta.yaw,
+			pitch: deviceYawPitchDelta.pitch,
+		});
+
+		const yawPitch = this.axes.get();
+
+		return {
+			yaw: yawPitch.yaw,
+			pitch: yawPitch.pitch,
+		};
 	}
 
 	getFov() {
 		return this.axes.get().fov;
 	}
 
+	getQuaternion() {
+		const pos = this.axes.get();
+
+		return this._deviceSensor.getCombinedQuaternion(pos.yaw);
+	}
+
+	shouldRenderWithQuaternion() {
```

Comment:
@WoodNeck 

Is there a code to use this method? also it is `isVR`?

Reply:
This is called on PanoImageRenderer to choose a rendering method.
It chooses renderWithQuaternion if gyroMode is set to VR.